opm/simulators/aquifers/AquiferInterface.hpp

```diff
@@ -78,15 +78,15 @@ class AquiferInterface
     int aquiferID() const { return this->aquiferID_; }
 
 protected:
-    bool co2store_() const
+    bool co2store_or_h2store_() const
     {
-        return ebos_simulator_.vanguard().eclState().runspec().co2Storage();
+        return ebos_simulator_.vanguard().eclState().runspec().co2Storage() || ebos_simulator_.vanguard().eclState().runspec().h2Storage();
```

Comment:
I think I'd create a small intermediate here and rewrite this as
```C++
const auto& rspec = ebos_simulator_.vanguard().eclState().runspec();
return rspec.co2Storage() || rspec.h2Storage();
```
instead.

Reply:
I agree, and have changed this.